<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AWorld Magicâ„¢</string>
  </property>
  <widget class="QLabel" name="titleLabel_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Project</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="mainFormLayout_">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameLineEdit_"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nameLabel_">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Project Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="vendorLabel_">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Design Vendor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="vendorLineEdit_"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="versionLabel_">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Design Version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="versionLineEdit_"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton_">
      <property name="text">
       <string>Create</string>
      </property>
      <property name="autoRepeatInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton_">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit_</tabstop>
  <tabstop>vendorLineEdit_</tabstop>
  <tabstop>versionLineEdit_</tabstop>
  <tabstop>okButton_</tabstop>
  <tabstop>cancelButton_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
